tests/xtro-sharpie/api-annotations-dotnet/macOS-HealthKit.ignore

```diff
@@ -0,0 +1,3 @@
+# intro reports them as not present
+!missing-selector! HKHealthStore::setWorkoutSessionMirroringStartHandler: not bound
```

Comment:
Is it available for Mac Catalyst (on Sonoma)?

It looks like the intention was to only make this available on iOS (only says available on iOS - maybe the author thought that would mean none of the other platforms):

```objective-c
/*!
 @property      workoutSessionMirroringStartHandler
 @abstract      Called when a session has started mirroring.
 @discussion    This property should always be assigned a value promptly after your app is launched,
                to ensure it is always observing for incoming mirrored workout sessions.
                If your app is not active when a mirrored session starts, it will be launched in the background and given a one-time
                permission to start a Live Activity from the background.
                The assigned block will be executed on an arbitrary background queue.
 */
@property (copy, nullable) void (^workoutSessionMirroringStartHandler)(HKWorkoutSession *mirroredSession) API_AVAILABLE(ios(17.0));
```

and the documentation agrees: "The system calls this block on the companion iPhone when someone starts a mirrored workout on Apple Watch."

So maybe we should remove this API from everywhere except iOS.

Reply:
lets do that, adding later is a better option than removing